ReSharper.FSharp/src/FSharp.Psi/src/CodeFormatter/FSharpCodeFormatterInfoProvider.cs

```diff
@@ -188,6 +192,37 @@ protected override void Initialize()
             When(true).Switch(settings => settings.NeverOutdentPipeOperators,
               When(false).Return(IndentType.Outdent | IndentType.External)))
           .Build();
+
+        Describe<FormattingRule>()
+          .Name("SpaceBetweenRecordBindings")
+          .Where(
+            Left()
+              .HasType(ElementType.RECORD_EXPR_BINDING)
+              .Satisfies((node, context) => ((IRecordExprBinding) node).Semicolon != null),
```

Comment:
It'd sad we currently have some of the semicolons inside the expressions (e.g. fields in record expressions have it inside) and some of them outside (record declarations and its fields, anon record field expressions). Would be great to try to unify it in FCS parser and our tree. Otherwise these rules probably won't work properly for some cases (e.g. anon record fields).